lib/internal/bootstrap/pre_execution.js

```diff
@@ -98,7 +101,11 @@ function patchProcessObject(expandArgv1) {
   if (expandArgv1 && process.argv[1] && !process.argv[1].startsWith('-')) {
     // Expand process.argv[1] into a full path.
     const path = require('path');
-    process.argv[1] = path.resolve(process.argv[1]);
+    try {
+      process.argv[1] = path.resolve(process.argv[1]);
+    } catch {
+      throw new ERR_INVALID_FILE_URL_PATH(process.argv[1]);
```

Comment:
```suggestion
```

Otherwise this is just exchanging an exception for an exception, which I don’t think would be particularly helpful, because it only loses information. (It might still fail at a later point, but I don’t think there’s anything we can do about that at this point then :confused:)

Reply:
@addaleax - makes sense, and followed your suggestions, PTAL.